buildbot-host/scripts/install-openvpn-build-deps-alpine.sh

```diff
@@ -11,9 +11,11 @@ ccache \
 cmake \
 cmocka-dev \
 curl \
+dbus-cpp-dev \
```

Comment:
This is not needed, but will need `dbus-dev` instead.

Reply:
Fixed. That said, Alpine builds continue to fail. I can report an issue about this if you want, but here's the build output from Buildbot:

```
[13/243] Compiling C++ object src/tests/getlaststatus.p/dbus_getlaststatus.cpp.o
ninja: job failed: ccache c++ -Isrc/tests/journal-log-parse.p -Isrc/tests -I../src/tests -I../openvpn3-core -I../vendor/asio/asio/include -Isrc -I../src -I. -I.. -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/gio-unix-2.0 -I/usr/include/blkid -I/usr/include/libmount -I/usr/include/libnl3 -I/usr/include/uuid -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -std=c++17 -O0 -g -Wno-non-virtual-dtor -DOVPN_TINYXML2_HAS_ERROR_NAME -DOVPN_TINYXML2_HAS_ERROR_STR -pthread -MD -MQ src/tests/journal-log-parse.p/misc_journal-log-parse-test.cpp.o -MF src/tests/journal-log-parse.p/misc_journal-log-parse-test.cpp.o.d -o src/tests/journal-log-parse.p/misc_journal-log-parse-test.cpp.o -c ../src/tests/misc/journal-log-parse-test.cpp
../src/tests/misc/journal-log-parse-test.cpp:19:17: error: 'Log' has not been declared
   19 | using namespace Log::Journald;
      |                 ^~~
../src/tests/misc/journal-log-parse-test.cpp:19:22: error: 'Journald' is not a namespace-name
   19 | using namespace Log::Journald;
      |                      ^~~~~~~~
../src/tests/misc/journal-log-parse-test.cpp: In function 'int main(int, char**)':
../src/tests/misc/journal-log-parse-test.cpp:23:5: error: 'Parse' was not declared in this scope
   23 |     Parse jrn;
      |     ^~~~~
../src/tests/misc/journal-log-parse-test.cpp:43:9: error: 'Parse' is not a class, namespace, or enumeration
   43 |         Parse::FilterType ft = {};
      |         ^~~~~
../src/tests/misc/journal-log-parse-test.cpp:46:13: error: 'ft' was not declared in this scope
   46 |             ft = Parse::FilterType::TIMESTAMP;
      |             ^~
../src/tests/misc/journal-log-parse-test.cpp:46:18: error: 'Parse' is not a class, namespace, or enumeration
   46 |             ft = Parse::FilterType::TIMESTAMP;
      |                  ^~~~~
../src/tests/misc/journal-log-parse-test.cpp:50:13: error: 'ft' was not declared in this scope
   50 |             ft = Parse::FilterType::LOGTAG;
      |             ^~
../src/tests/misc/journal-log-parse-test.cpp:50:18: error: 'Parse' is not a class, namespace, or enumeration
   50 |             ft = Parse::FilterType::LOGTAG;
      |                  ^~~~~
../src/tests/misc/journal-log-parse-test.cpp:54:13: error: 'ft' was not declared in this scope
   54 |             ft = Parse::FilterType::SESSION_TOKEN;
      |             ^~
../src/tests/misc/journal-log-parse-test.cpp:54:18: error: 'Parse' is not a class, namespace, or enumeration
   54 |             ft = Parse::FilterType::SESSION_TOKEN;
      |                  ^~~~~
../src/tests/misc/journal-log-parse-test.cpp:58:13: error: 'ft' was not declared in this scope
   58 |             ft = Parse::FilterType::OBJECT_PATH;
      |             ^~
../src/tests/misc/journal-log-parse-test.cpp:58:18: error: 'Parse' is not a class, namespace, or enumeration
   58 |             ft = Parse::FilterType::OBJECT_PATH;
      |                  ^~~~~
../src/tests/misc/journal-log-parse-test.cpp:62:13: error: 'ft' was not declared in this scope
   62 |             ft = Parse::FilterType::SENDER;
      |             ^~
../src/tests/misc/journal-log-parse-test.cpp:62:18: error: 'Parse' is not a class, namespace, or enumeration
   62 |             ft = Parse::FilterType::SENDER;
      |                  ^~~~~
../src/tests/misc/journal-log-parse-test.cpp:66:13: error: 'ft' was not declared in this scope
   66 |             ft = Parse::FilterType::INTERFACE;
      |             ^~
../src/tests/misc/journal-log-parse-test.cpp:66:18: error: 'Parse' is not a class, namespace, or enumeration
   66 |             ft = Parse::FilterType::INTERFACE;
      |                  ^~~~~
../src/tests/misc/journal-log-parse-test.cpp:73:9: error: 'jrn' was not declared in this scope
   73 |         jrn.AddFilter(ft, value);
      |         ^~~
../src/tests/misc/journal-log-parse-test.cpp:73:23: error: 'ft' was not declared in this scope
   73 |         jrn.AddFilter(ft, value);
      |                       ^~
../src/tests/misc/journal-log-parse-test.cpp:76:5: error: 'LogEntries' was not declared in this scope
   76 |     LogEntries l = jrn.Retrieve();
      |     ^~~~~~~~~~
../src/tests/misc/journal-log-parse-test.cpp:78:18: error: 'l' was not declared in this scope
   78 |     std::cout << l.GetJSON() << std::endl;
      |                  ^
ninja: subcommand failed
INFO: autodetecting backend as ninja
INFO: calculating backend command to run: /usr/bin/ninja -C /buildbot/alpine-openvpn3-linux-openssl/build/_builddir
program finished with exit code 1
elapsedTime=157.954097
```

Reply:
I'll resolve this comment as the build failure is unrelated to the actual package name.